composer.json

```diff
@@ -22,10 +22,12 @@
         "phpcs": "phpcs -n --ignore=vendor --extensions=php .",
         "phpcbf": "phpcbf -n --ignore=vendor --extensions=php .",
         "phplint": "parallel-lint -j 10 --exclude vendor .",
+        "selftest": "@php tests/functional/runner.php",
```

Comment:
klidně by to mohlo bejt `tests-functional` místo `selftest`?

Reply:
Done. True.